client/coral-admin/src/graphql/queries/index.js

```diff
@@ -62,13 +63,21 @@ export const loadMore = (fetchMore) => ({limit, cursor, sort, tab, asset_id}) =>
       asset_id
     },
     updateQuery: (oldData, {fetchMoreResult:{comments}}) => {
-      return {
+      const updatedData = {
         ...oldData,
         [tab]: [
           ...oldData[tab],
           ...comments
         ]
       };
+
+      // if we're not in the all tab, put the new comments in there anyway
+      // this way the acceptComment and rejectComment mutations don't break
```

Comment:
Going through the code again I realised, that adding this code here is the wrong approach, and tries to work around the fact that `acceptComment` and `rejectComment` have some issues and need to be implemented differently.